### PR TITLE
[WIP] Dashboard

### PR DESCRIPTION
**Pull request by engineSound**
This is the latest UI development for dashboard without backend integration. Alva's team is working on backend REST API. 

Once REST API is ready, dashboard.js will make ajax REST call and process data to present to a table in dashboard page. 

The critical changes is in the template dashboard.html which uses template include. And also the routing part in the urls.py and theme/views.py.

This is to let you be aware what I am getting at.

### CODE REVIEW

File: hydroshare/urls.py [hunk 1]
```diff
@@ -131,7 +134,7 @@
     # one out.
 
     # url("^$", direct_to_template, {"template": "index.html"}, name="home"),
-    url(r"^tests/$", direct_to_template, {"template": "tests.html"}, name="tests"),
+    # QUNIT_TESTS_OFF
```
Comment (cuahsimarko) [inline hydroshare/urls.py]: what is the reason for this change?

Comment (engineSound) [reply hydroshare/urls.py]: good catch. This line shouldn't be changed. 
It is a puzzle how I let this line changed.

**Comment by engineSound** [comment]
Combined UI and backend code. Review in one place. #3301 

Thus, closing this.

**Comment by cuahsimarko** [comment]
@engineSound does this mean you're going to open another PR?

**Comment by engineSound** [comment]
@cuahsimarko #3301 is the place to review both backend and frontend code.
